great_expectations/render/view/templates/markdown_page.j2

```diff
@@ -0,0 +1,10 @@
+
+# Validation Results
```

Comment:
Not necessarily a deal breaker since this is sort of a partner-centric PR, but I think we should either make this more general (i.e. not just for validation result pages, like the html page.j2 template is), or just make the template more specific (rename to something like `markdown_validation_results_page.j2`).

Reply:
Makes total sense, I like the renaming idea to add specificity.